doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -237,19 +237,19 @@ construct differs from the latter as follows.
 
 
 The ``let:`` construct is just (more legible) notation for the primitive
-Gallina expression :n:`match @term with @pattern => @term end`.
+Gallina expression :n:`match @term with @pattern => @term__body end`.
 
 The |SSR| destructuring assignment supports all the dependent
 match annotations; the full syntax is
 
 .. prodn::
    term += let: @pattern {? as @ident} {? in @pattern__ind} := @term {? return @term__ret} in @term__body
 
-
-When the ``as`` and ``return`` keywords are both present, then :token:`ident` is bound
-in both :n:`@term__ret` and :n:`@term__body`; variables in :n:`@pattern__ind` are bound
-only in :n:`@term__ret`, and other variables in :token:`pattern` are bound only in
-:n:`@term__body`, however.
+Similarly, this dependent ``let:`` construct is just notation for :n:`match
+@term {? as @ident } {? in @pattern__ind } {? return @term__ret } with @pattern
+{? as @ident } => @term__body end`. In particular, :n:`@ident` is used both for
+dependent pattern matching and for aliasing the pattern (see
+:ref:`aliasing-subpatterns`).
```

Comment:
Worth saying ":token:`pattern` binds variables in :n:`@term__else` and not in :n:`@term__then`."?